OctopusDSC/DSCResources/cTentacleAgent/cTentacleAgent.psm1

```diff
@@ -389,8 +389,10 @@ function Install-Tentacle
 
     $actualTentacleDownloadUrl = Get-TentacleDownloadUrl $tentacleDownloadUrl $tentacleDownloadUrl64
 
-    mkdir "$tentacleHomeDirectory" -ErrorAction SilentlyContinue
-
+    if(-not (Test-Path $tentacleHomeDirectory))
+    {
+        New-Item -Path "$tentacleHomeDirectory" -ItemType Directory 
```

Comment:
Maybe a log line saying what its doing (both to be more explicit, but also to match what you're doing below?)